docs/core/install/runtime.md

```diff
@@ -81,6 +84,23 @@ dotnet-install.ps1 -Channel 3.1 -Runtime aspnetcore
 > [!NOTE]
 > The command above installs the ASP.NET Core runtime for maximum compatability. The ASP.NET Core runtime also includes the standard .NET Core runtime.
 
+## Download and manually install
+
+First [download](#all-net-core-downloads) a .NET Core binary release. Then create a directory to install to, for example `%USERPROFILE%\dotnet`. And finally extract the downloaded zip file into that directory.
+
+.NET Core installed in this way will not be picked up by default, you have to explicitly choose to use it. To do so, change the environment variables with which an application is started:
```

Comment:
A few things:
* I think "picked up" could be clarified. Edited arrangement a bit for active voice and removed comma splice:
  ```suggestion
  By default, .NET Core CLI commands and apps will not use .NET Core installed in this way. You have to explicitly choose to use it. To do so, change the environment variables with which an application is started:
  ```
* I think this should clarify that these are console (not powershell) commands. Or can it offer a switch to show either syntax?
* Seems it should also mention that it will only take effect in the terminal they're typed into. The Linux side is more detailed about env vars... how to set them permanently, for example. Might need similar detail on Windows?

Reply:
On Windows it's very unlikely users will want to install into a custom location and set that as the "global" install (unlike Linux, where for some distros that's the only way to do it right now). So I intentionally omitted the part about global setup.

Also in 3.0 the global registration can now be done via registry/conf files - which is a different type of change. Not sure it should be mentioned here though - as that's mostly intended for installers (and advanced usage).